mmv1/templates/terraform/examples/region_backend_service_ip_address_selection_policy.tf.tmpl

```diff
@@ -0,0 +1,18 @@
+resource "google_compute_region_backend_service" "{{$.PrimaryResourceId}}" {
+  name                            = "{{index $.Vars "region_backend_service_name"}}"
+  region                          = "us-central1"
+  health_checks                   = [google_compute_region_health_check.default.id]
```

Comment:
Looks like this should be `google_compute_region_health_check. health_check.id`, then hopefully tests will pass

Reply:
Thanks, should be fixed. I figured out how to run manually and it's passing in my repo now also.
```
$ make testacc TEST=./google/services/compute TESTARGS='-run=TestAccComputeRegionBackendService_regionBackendServiceIpAddressSelectionPolicyExample'
TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test ./google/services/compute -v -run=TestAccComputeRegionBackendService_regionBackendServiceIpAddressSelectionPolicyExample -timeout 240m -ldflags="-X=github.com/hashicorp/terraform-provider-google/version.ProviderVersion=acc"
=== RUN   TestAccComputeRegionBackendService_regionBackendServiceIpAddressSelectionPolicyExample
=== PAUSE TestAccComputeRegionBackendService_regionBackendServiceIpAddressSelectionPolicyExample
=== CONT  TestAccComputeRegionBackendService_regionBackendServiceIpAddressSelectionPolicyExample
--- PASS: TestAccComputeRegionBackendService_regionBackendServiceIpAddressSelectionPolicyExample (40.34s)
PASS
ok      github.com/hashicorp/terraform-provider-google/google/services/compute  40.804s
```